constants/appLinks.ts

```diff
@@ -52,6 +52,7 @@ const LEARN_START_LEARNING: GeneralLink = {
 enum StartLearningUrl {
   Introduction = 'https://learn.qiskit.org/course/introduction',
   BasicsQuantumInformation = 'https://learn.qiskit.org/course/basics',
+  NearTermAlgorithmDesign = 'https://learn.qiskit.org/course/algorithm-design',
```

Comment:
Thanks for the PR @pandasa123 - I'm good w/ everything here... though my only question is, when will this be ready?

I believe it's waiting for [this PR from Platypus](https://github.com/Qiskit/platypus/pull/1965)... 

Are you saying that we're okay to _dark-launch_ this landing page w/o the course page being ready, so we satisfy requirements for the badge?